scripts/load-environment-from-secrets.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+./check-env-variables.sh \
+  "REGION" \
+  "ENVIRONMENT" \
+  "AWS_SECRET_ACCESS_KEY" \
+  "AWS_ACCESS_KEY_ID"
+
+REGION=us-east-1
+content=$(aws secretsmanager get-secret-value --region ${REGION} --secret-id "${ENVIRONMENT}_deploy" --query "SecretString" --output text)
```

Comment:
I was thinking more along the lines of something like this, but have no issues with leaving as you have it. Of course with this, each script that calls this one would need to pass the full secret name as a parameter.

```suggestion
SECRET_NAME=$1

REGION=us-east-1
content=$(aws secretsmanager get-secret-value --region ${REGION} --secret-id "${SECRET_NAME}" --query "SecretString" --output text)
```